z3/src/ast.rs

```diff
@@ -642,6 +643,7 @@ impl<'ctx> Int<'ctx> {
     binop!(le, Z3_mk_le, Bool<'ctx>);
     binop!(gt, Z3_mk_gt, Bool<'ctx>);
     binop!(ge, Z3_mk_ge, Bool<'ctx>);
+    binop!(_eq, Z3_mk_eq, Bool<'ctx>);
```

Comment:
The `_eq` method is already defined on the `Ast` trait - is there a reason it needs to be separately defined on `Int`?  I believe this implementation is the same one that's already provided by `Ast`.

Reply:
No, I overlooked the definition in `Ast`.